docs/guides/implementation-guides/how-to-implement-dependency-injection.md

```diff
@@ -20,14 +20,16 @@ public MainViewModel(MyService myService)
 }
 ```
 
-You could solve that by instanciating `MyService` and passing as a parameter everytime you need a `MainViewModel` instance, like this:
+You could solve that by instantiating `MyService` and passing as a parameter everytime you need a `MainViewModel` instance, like this:
```

Comment:
It's long as you said but as it will be helpful as it's explicitly listing the problem instead of just alluding to them. For me both are correct it's just depends of the tone of voice and the target audience of the documentation.

Reply:
I've been able to make it easier to read by using list. I think it's a nice addition, thanks for the feedback